src/Writer/XlsExcelWriter.php

```diff
@@ -0,0 +1,187 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Exporter\Writer;
+
+/**
+ * .xls writer bridge for library `phpoffice/phpexcel`
+ * @see https://github.com/PHPOffice/PHPExcel
+ *
+ * @package Exporter\Writer
+ * @author Lukáš Brzák <lukas.brzak@email.cz>
+ */
+class XlsExcelWriter implements TypedWriterInterface
```

Comment:
Why you think this class should be final? If somebody wants to edit e.g. styles, charts and so on, with final is not possible.